setup.py

```diff
@@ -31,6 +29,11 @@ def get_version():
     'six>=1.11.0,<2.0',
     'protobuf>=3.0.0,<4.0',
     'pyRFC3339>=1.0,<2.0',
+    'ipaddress;python_version<="2.7"',
```

Comment:
Replace <="2.7" with <"3.0" ? It seem more in line with the original check. What do you think?

Reply:
sure thing, I agree with your reasoning -- even if it's _unlikely_ there will be a 2.8 you never know!